### PR TITLE
AUTO_GENERATED files removal

### PR DESCRIPTION
**Pull request by Kiel-SFU**
There's some strangeness with one of the CSV's that I'll try to ask you about on October 20

### CODE REVIEW

File: scripts/SpecifiedDemandProfile.py [hunk 1]
```diff
@@ -41,10 +42,13 @@ def main():
     ###########################################
 
     #generate master load dataframe
-    dfLoad = getLoadValues()
+    dfLoad = functions.getLoadValues()
 
-    #save load dataframe for use in other scripts
-    dfLoad.to_csv('../dataSources/ProvincialHourlyLoads_TimeAdjusted_AUTO_GENERATED.csv', index=False)
+    ###########################################
```
Comment (trevorb1) [inline scripts/SpecifiedDemandProfile.py:47]: Lets just remove this comment block. Continued from our conversation earlier, I think it was my fault on saying this auto_gen file was created in this script.

File: .gitignore [hunk 2]
```diff
@@ -3,7 +3,7 @@ dataSources/*_RAW.csv
 dataSources/*_RAW.xlsx
 scripts/.vscode/
 src/.snakemake/
-scripts/dag*
+src/dag*
 results/
 scripts/.ipynb_checkpoints/
 dataSources/UpdateDemand/
```
Comment (trevorb1) [inline .gitignore:9]: There is a file called `scripts/__pycache__/functions.cpython-39.pyc` in this pull request. Can we add this to the GitIgnore, or is it needed?

File: src/snakefile [hunk 3]
```diff
@@ -118,7 +120,7 @@ rule lpFile:
 ## INTERMEDIATE RULES
 #####################################
 
-rule ConfigFile:
+rule config:
```
Comment (trevorb1) [inline src/snakefile:123]: Lets change this to `configRule` (or `ConfigRule`, whatever convention you are using). When we build in the configuration file to the workflow, we import it using the `config` or `configFile` keyword I think. So just improves clarity (my original of name of `ConfigFile` for the rule probably wasnt the best option either)

File: scripts/Regionalization.py [hunk 4]
```diff
@@ -1,32 +0,0 @@
...
-    
-    for region in regions:
-        dataOut.append(region)
-
-    #write to a csv
-    dfOut = pd.DataFrame(dataOut,columns=['VALUE'])
-    dfOut.to_csv('../src/data/Canada/REGION.csv', index=False)
-
-if __name__ == "__main__":
-    main()
```
Comment (trevorb1) [inline scripts/Regionalization.py:32]: This is just generated by the `config.py` file now right?

**Review by trevorb1 — CHANGES_REQUESTED** [review]

File: scripts/Regionalization.py (hunk 4 above)
Comment (Kiel-SFU) [reply scripts/Regionalization.py:32]: No, I think we removed this script because it was unused and ungenerated.

Comment (trevorb1) [reply scripts/Regionalization.py:32]: Ahh... This is super old, before we built the regions into the technology names. We used to have the regions (CANWS, CANMW, ect...) under the region set, but now everything is just classified as NAMERICA. We are good to delete

**Pull request merged**